src/Components/Fsi.fs

```diff
@@ -320,7 +320,6 @@ module Fsi =
         )
 
     let private start () =
-        fsiOutput |> Option.iter (fun n -> n.dispose())
```

Comment:
all of this state management (cleanup of old terminal, tracking of new values) is handled in the `handleOpenTerminal` event now, which is fired after this terminal is created.